### DISCUSSION TITLE
Use vscode cmd+d

### DISCUSSION THREAD

**Discussion by tellcodes** [vscode-neovim/vscode-neovim#1648] [General]
Hi!

I haven't found anything that meet my needs when it comes to multi-cursor editing. I have seen this [thread](https://github.com/vscode-neovim/vscode-neovim/issues/409) and this [plugin](https://github.com/vscode-neovim/vscode-multi-cursor.nvim).

Is it possible to get VScodes native multi-cursor functionality (with cmd+D) to work as normal while having the vscode-neovim extension enabled?

**Comment by xiyaowong** [comment]
https://github.com/vscode-neovim/vscode-multi-cursor.nvim#wrapped-vscode-commands
